crawler/elasticClient.go

```diff
@@ -29,6 +29,31 @@ func ElasticClientFactory(URL, user, password string) (*elastic.Client, error) {
 	return client, nil
 }
 
+// ElasticAliasUpdate update the Alias to the index.
+func ElasticAliasUpdate(index, alias string, elasticClient *elastic.Client) error {
+	// Remove old aliases.
+	res, err := elasticClient.Aliases().Index("_all").Do(context.Background())
```

Comment:
this remove all aliases in general? Maybe we can remove only the `publiccode`alias

Reply:
I think my comment is wrong.
`// Remove old aliases.` should be `// Retrieve all the aliases.` and then remove the `publiccode` alias.